react-native-xmpp.podspec

```diff
@@ -0,0 +1,14 @@
+Pod::Spec.new do |s|
+  s.name         = "react-native-xmpp"
+  s.version      = "0.2.1"
+  s.license      = "MIT"
+  s.homepage     = "https://github.com/aksonov/react-native-xmpp"
+  s.authors      = { 'Marc Shilling' => 'marcshilling@gmail.com' }
```

Comment:
I was taking a step at adding a podspec so you can include this dependency more easily in a cocoapods managed project, but it seems to require quite a lot additional changes, sadly